### ISSUE TITLE
[0.19.0] Loading resources on Windows fails

### ISSUE THREAD

**Issue by YuukanOO** [snipsco/snips-nlu#750]
Since the new release, loading resources is not possible on Windows.

In fact, on Windows, since the linking of resources via `python -m snips_nlu download en` fail, I used to use `load_resources('snips_nlu_en')`.

Now, since `load_resources` is also called in the `fit` method using the dataset language, it complains about the language resource 'en' not being found.

I also tried something like this:

```python
from snips_nlu import SnipsNLUEngine, load_resources
import json

with open('training.json') as f:
  data = json.load(f)

r = load_resources('snips_nlu_en')

engine = SnipsNLUEngine(resources=r)

engine.fit(data)
```

but it doesn't work. Any ideas on how to make it work under Windows?

**Comment by adrienball** [comment]
Hi @YuukanOO ,
Thanks for reporting.
The resources are now lazily loaded when calling `fit`, using `load_resource(language)`.
You can avoid this lazy behavior and load the resources beforehand and provide it when you create a `SnipsNLUEngine`, which is exactly what you tried in the code sample that you provided, and it should have worked.
However there is a bug that I just found (thank you), in the logic that checks whether or not each sub-unit must reload the resources, resulting in an unnecessary attempt to load resources.
I'm currently working on a hotfix, and then we will quickly provide a new release.
Thanks again for reporting.
Cheers

**Comment by adrienball** [comment]
@YuukanOO this should be addressed by #751 
Can you upgrade to `0.19.1` and confirm that it is fixed?
Cheers

**Comment by YuukanOO** [comment]
It's fixed! Thanks for the quick answer 😉
